.github/workflows/backstop.yml

```diff
@@ -0,0 +1,103 @@
+name: Backstop Tests
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  run-backstop-tests:
+    if: github.event.label.name == 'run backstop'
+    runs-on: ubuntu-latest
+    container:
+      image: compucorp/civicrm-buildkit:1.0.0-chrome
+
+    env:
+      CIVICRM_EXTENSIONS_DIR: site/web/sites/all/modules/civicrm/tools/extensions
+      DRUPAL_THEME_DIR: site/web/sites/all/themes
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+      - name: Config mysql database as per CiviCRM requirement
+        run: echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" | mysql -u root --password=root --host=mysql
+
+      - name: Config amp
+        run : |
+          amp config:set --db_type=mysql_dsn --mysql_dsn='mysql://root:root@mysql:3306' --httpd_type=apache24 --httpd_restart_command='sudo /usr/sbin/apache2ctl graceful' --perm_type=worldWritable --hosts_type=file
+          echo "IncludeOptional /github/home/.amp/apache.d/*.conf" >> /etc/apache2/apache2.conf
+          /usr/sbin/apache2ctl restart
+
+      - name: Build Drupal site
+        run: |
+          civibuild create drupal-clean --civi-ver 5.24.6 --web-root $GITHUB_WORKSPACE/site
+          chmod -R 777 /__w/uk.co.compucorp.civicase/uk.co.compucorp.civicase
+
+      - name: Installing CiviCase and Shoreditch
+        working-directory: ${{ env.CIVICRM_EXTENSIONS_DIR }}
+        run: |
+          git clone https://github.com/compucorp/uk.co.compucorp.civicase.git
+          git clone --depth 1 https://github.com/civicrm/org.civicrm.shoreditch.git
```

Comment:
@deb1990 what are the chances of a change in civicase to depend on a change in shoreditch (i.e. something that hasn't been merged to master yet)? If there are any chances of this happening, how to you plan to test that using this workflow?

Reply:
@davialexandre As of now, that chance is really low practically. I dont even remember when that happened last time in recent future.
While I dont want to overcomplicate the PR for now, but I think the ability to specifiy Shoreditch branch using PR comment can be added in future. Also we can further improve it by mentioning the civicase branch in the comments too(so instead of master branch, it would compare with the branch mentioned in the comments).